media/css/firefox/whatsnew/whatsnew-95-en.scss

```diff
@@ -115,15 +115,10 @@ $image-path: '/media/protocol/img';
         color: $color-white;
     }
 
-    .wnp-main-title,
-    .wnp-footer-title {
+    .wnp-main-title {
         color: get-theme('title-text-color-inverse');
     }
 
-    .c-utilities {
```

Comment:
I don't think we needed to remove this?

Reply:
realized that the link color in the design uses the normal blue color, not white like what is used in DE design - so i removed it.

Reply:
I think this is needed for the dark theme styling else there's a link contrast issue: 
![image](https://user-images.githubusercontent.com/400117/144208223-6a623573-ae9c-4d09-8c25-87f0782a78e9.png)